site/ko/blog/new-in-devtools-95/index.md

```diff
@@ -6,7 +6,8 @@ authors:
 date: 2021-09-20
 updated: 2021-09-20
 description:
-  "New CSS length authoring tools, hide issues in the Issues tab and improved the display of properties more."
+  "
+  새 CSS 길이 작성 도구, 문제 탭에서 문제를 숨기고 속성 표시 개선 등"
```

Comment:
How about this?
"새 CSS 길이 작성 도구, 문제 탭에서 문제 숨김 및 개선된 속성 표시 등"
And here's one line beak looking unnecessary
(Let me ask @chybie about the context for just in case)

Reply:
Typo on my side, should be "New CSS length authoring tools, hide issues in the Issues tab, improved the display of properties and more."

Reply:
I'd like to suggest this,
 "새 CSS 길이 작성 도구, 문제 탭에서 문제 숨김, 개선된 속성 표시 및 다른 기능들"

Reply:
It's a bit unclear what '개선된' describes whether it's 속성 or 표시.

Suggestion:
새 CSS 길이 작성 도구, 문제 탭에서 문제 숨기기 기능, 속성 표시 방법 개선 및 다른 새로운 기능들.